src/components/swipe-action/swipe-action.tsx

```diff
@@ -40,13 +44,15 @@ export type SwipeActionProps = {
   closeOnTouchOutside?: boolean
   closeOnAction?: boolean
   children: ReactNode
+  stopPropagation?: PropagationEvent[]
 } & NativeProps<'--background'>
 
 const defaultProps = {
   rightActions: [] as Action[],
   leftActions: [] as Action[],
   closeOnTouchOutside: true,
   closeOnAction: true,
+  stopPropagation: ['click'],
```

Comment:
默认不能阻止的……不然会导致 break change 了

```suggestion
  stopPropagation: [],
```

（文档也要再调整下～

Reply:
ok